src/object_store/src/object/s3.rs

```diff
@@ -702,6 +726,24 @@ impl S3ObjectStore {
             }
         }
     }
+
+    #[inline(always)]
+    fn get_retry_strategy() -> Map<Take<ExponentialBackoff>, fn(Duration) -> Duration> {
+        ExponentialBackoff::from_millis(DEFAULT_RETRY_INTERVAL)
```

Comment:
Does it means that any request which exceed 20ms would send another request ?
20ms is too small for s3 storage. 
When we read a large object ,the latency could be several hundred mills

Reply:
No, only timeout error occurs will will retry, see `should_retry`.